components/Organization/New/Step2DescribeOrganization.vue

```diff
@@ -297,22 +297,22 @@ function checkBusinessId<T, K extends KeysOfUnion<T>, V extends (string | undefi
     }
 
     if (sum % 10 === 0) return null
-    return message || t('Ce numéro SIRET est invalide.')
+    return t('Ce numéro SIRET est invalide.')
   }
 }
 
 const { form, formInfo, getFirstError, getFirstWarning, touch, validate } = useForm(organization, {
-  business_number_id: [checkBusinessId(checkOrga)],
+  business_number_id: [checkBusinessId(), unique(siret => `/api/1/organizations/?business_number_id=${cleanSiret(siret)}`, t('Une organisation avec ce SIRET existe déjà sur {site}', { site: config.public.title }))],
```

Comment:
The `unique` rules should be added only if the prop `type` is of type `create`

Reply:
Good catch, fixed in 40e290a